tests/test_core/test_area/test_utm.py

```diff
@@ -72,7 +72,7 @@ def test_get_grid_caching(storage, config):
 
     grid1 = area_manager.get_grid()
     # the second call should not create a new split
-    with mock.patch.object(UtmZoneAreaManager, "_create_new_split") as creation_mock:
+    with patch.object(UtmZoneAreaManager, "_create_new_split") as creation_mock:
```

Comment:
Is it necessary to test each `AreaManager` separately, or could you just test the `BaseAreaManager` functionality for caching?